pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -376,14 +382,15 @@ stdenv.mkDerivation (finalAttrs: {
      *  Licensing flags
      */
     (enableFeature withGPL "gpl")
-    (enableFeature withGPLv3 "version3")
+    (enableFeature withVersion3 "version3")
     (enableFeature withUnfree "nonfree")
     /*
      *  Build flags
      */
-    # On some ARM platforms --enable-thumb
-    "--enable-shared"
-    "--enable-pic"
+    (enableFeature withStatic "static")
+    (enableFeature withShared "shared")
+    (enableFeature withPic "pic")
+    (enableFeature withThumb "thumb")
```

Comment:
Have you verified that these work?

Reply:
Tested static and shared (works if one or both are enabled, static only tested on headless), chaining pic works with static on shared it needs to enabled other wise the build fails, can not test enabling static as i do not own a 32-bit arm machine with thumb support